src/components/Search.js

```diff
@@ -30,8 +30,13 @@ class Search extends Component {
             className="searchValue" // onClick={deleteSearch}
           />
         </div>
-        {danceFilter.map((c) => (
-          <ListItem key={c.Id} move={c.Move} creator={c.Creator} Id={c.Id} />
+        {danceFilter.map((filteredDanceMove) => (
```

Comment:
I feel that name **filteredDanceMove** is little too much, because we have array danceFiltered. Better name in my opinion would be just **move**. That danceFilter sound not so good. That name could be filteredDanceMoves.

Reply:
If you are ok with your names, I can merge 😎

Reply:
I named it like that because when we went through this code with others one person suggested that, so they could understand what is happening there. :)
If we name it just to move it can be confusing move={move.Move}... 
Can maybe other gives opinions as well? :)

Reply:
Ok. I will accept. 😃